src/ic-certified-assets/src/lib.rs

```diff
@@ -687,8 +687,8 @@ fn check_url_decode() {
 fn redirect_to_url(host: &str, url: &str) -> Option<String> {
     if let Some(host) = host.split(':').next() {
         let host = host.trim();
-        if let Some(base) = host.strip_suffix(".raw.ic0.app") {
-            return Some(format!("https://{}.ic0.app{}", base, url));
+        if let Some(base) = host.strip_suffix("raw.ic0.app") {
```

Comment:
What would you think about handling both of these cases explicitly?
- equal to `raw.ic0.app`
- ends with `.raw.ic0.app`

Another form of the url is, for example, `https://n4nzw-wqaaa-aaaai-aa7yq-cai.ic0.app`.  I don't know if the last three characters in a principal are constrained such that they cannot be `raw`.